platformio.ini

```diff
@@ -12,7 +12,7 @@
 platform = espressif8266
 framework = arduino
 board_f_cpu = 160000000L
-build_flags = !python scripts/firmware_version.py -Wall
+build_flags = !python scripts/firmware_version.py -Wall -DARDUINOJSON_ENABLE_PROGMEM=1
```

Comment:
We do not need this. We just need to make sure that `WString.h` is included before `ArduinoJson.h`. You can put the include statements in different blocks separated by a new line, then clang-format will not change their order:
```cpp
#include <WString.h>

#include <ArduinoJson.h>
```
Then we also do not need ugly workarounds like in `ConfigWebServer.cpp`. Please clean them all up.

Reply:
It crashes without that when you use PROGMEM as keys.